package.json

```diff
@@ -4,25 +4,41 @@
   "description": "",
   "main": "build/index.js",
   "types": "build/index.d.ts",
+  "keywords": [
+    "config",
+    "typescript",
+    "node-config",
+    "environment-variables",
+    "env",
+    "environment",
+    "settings"
+  ],
+  "author": "Stephen Haberman",
+  "license": "MIT",
+  "homepage": "https://github.com/stephenh/ts-app-env/#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stephenh/ts-app-env.git"
+  },
+  "bugs": "https://github.com/stephenh/ts-app-env/issues",
```

Comment:
I always try to run `npm repo <package>` to jump to github. this will allow that.